web/src/engine/events/src/keyEventSource.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { EventEmitter } from "eventemitter3";
-import { type KeyEvent, type RuleBehavior } from "@keymanapp/keyboard-processor";
+import { type KeyEvent, type RuleBehaviorInterface } from "@keymanapp/keyboard-processor";
 
-export type KeyEventResultCallback = (result: RuleBehavior, error?: Error) => void;
+export type KeyEventResultCallback = (result: RuleBehaviorInterface, error?: Error) => void;
 export type KeyEventHandler = (event: KeyEvent, callback?: KeyEventResultCallback) => void;
```

Comment:
Performing a text-search across the repo... these two exported types are only used by `VisualKeyboard`, within `engine/osk`.  They aren't even used here!

Just relocate them, and we'll have no need for the added `RuleBehaviorInterface` type.  Simpler and less messy that way.